test_files/comments/comments.tsickle.ts

```diff
@@ -14,13 +14,15 @@ export1: string;
   nodoc2: number;
   /// non js-doc comment
   nodoc3: number;
-  /** inline jsdoc comment without type annotation */
-  jsdoc1: number;
-  /**
-   * multi-line jsdoc comment without
-   * type annotation.
-   */
-  jsdoc2: number;
+/**
+ * inline jsdoc comment without type annotation
```

Comment:
Something in TS changed their comment parsing? Might be interesting to track down what the change was, I recall you had some requests for them to improve it